os/board/common/Kconfig

```diff
@@ -134,3 +134,199 @@ config AUTOMOUNT_ROMFS
 	depends on FS_ROMFS
 	---help---
 		If enabled, mount romfs partition automatically at boot.
+
+config BOARD_ENTROPY_POOL
+        bool "Enable Board level storing of entropy pool structure"
+        default n
+        depends on CRYPTO_RANDOM_POOL
+        ---help---
+                Entropy pool structure can be provided by board source.
+                Use for this is, for example, to allocate entropy pool
+                from special area of RAM which content is kept over
+                system reset.
+
+config BOARD_INITRNGSEED
+        bool "Enable Board level initial seeding of entropy pool RNG"
+        default n
+        depends on CRYPTO_RANDOM_POOL
+        ---help---
+                If enabled, entropy pool random number generator will call
+                board_init_rndseed() upon initialization. This function
+                can then provide early entropy seed to the pool through
+                entropy injection APIs provided at 'nuttx/random.h'.
+
+config BOARDCTL
```

Comment:
As I left before, BOARDCTL looks duplicated. Please confirm it again.

Reply:
It is not duplicated

Reply:
>> I think this and belows are in os/board/Kconfig. I think they are duplicated. Could you confirm?

This is what I left before. I think you didn't check that yet. Please check it again and let me know why you think it's not duplicated.

Reply:
Okay, thank you I get it. In both the files it is with different name but doing same work( lib_boardctl and boardctl). I am changing it

Reply:
Some sub-configs are perfectly the same. Please check sub-configs as well. (I mean BOARDCTL_XXXXX configs)